src/main/resources/data.properties

```diff
@@ -63,7 +63,7 @@ log.all.requests.responses=false
 # Identity
 identity.feature.toggle=false
 identity.realm=backbase
-identity.client=hybrid-flow
+identity.client=admin-cli
```

Comment:
Is this the right client to use?  I chose this option as it was one that works, however other clients exist.

Reply:
```suggestion
identity.client=bb-tooling-client
```

Reply:
The bb-tooling-client is what we use for getting a token to perform requests.
bb-web-client is for web login.
The rest is generally not used in the Backbase realm.